src/nimble_data_manager.h

```diff
@@ -209,11 +193,6 @@ namespace nimble {
     //! Map from global node it to local node id
     std::map<int, int> global_node_id_to_local_node_id_;
 
-#ifdef NIMBLE_HAVE_UQ
-    //struct setting up the macroscale UQ model
-    UqModel uq_model_;
```

Comment:
Maybe make this a shared_ptr which is optionally nullptr? Avoid the need for ifdefs, just don't construct it at runtime when UQ isn't enabled.